packages/desktop-client/src/components/select/RecurringSchedulePicker.js

```diff
@@ -172,7 +172,9 @@ function SchedulePreview({ previewDates }) {
   } else {
     content = (
       <View>
-        <Text style={{ fontWeight: 600 }}>Upcoming dates</Text>
+        <Text style={{ color: theme.tableText, fontWeight: 600 }}>
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏why do we need to set the color here?